pkg/controllers/monitoring/monitoring-stack/components.go

```diff
@@ -172,6 +172,8 @@ func newPrometheus(
 				RemoteWrite:               config.RemoteWrite,
 				ExternalLabels:            config.ExternalLabels,
 				EnableRemoteWriteReceiver: config.EnableRemoteWriteReceiver,
+				NodeSelector:              config.NodeSelector,
+				Tolerations:               config.Tolerations,
```

Comment:
@bng0y , Like @simonpasquier suggested, lets make this global and apply the same to all resources that ObO creates.